### ISSUE TITLE
A11y_Visual Studio Code_HomePage_Reflow: All options in command bar are not clearly visible at 400% zoom

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/vscode-docs#4517]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11y_Visual Studio Code#A11ySev2#A11yTCS#A11yWCAG2.1#Chrome#DesktopWeb#Win10#WCAG1.4.10#Reflow#Zoom#Visual Studio Code Services#VisualStudioCode-Web-Apr2021

### Environment details:
Application Name: Visual Studio Code
URL:[Visual Studio Code](https://code.visualstudio.com/)
Windows Version: Version 20H2 (OS Build 19042.928)
Browser: Chrome Version 90.0.4430.85 (Official Build) (64-bit)

### Repro Steps:

1. Change the display resolution to 1280*1024. Zoom the page to 400%
2. Hit [Visual Studio Code](https://code.visualstudio.com/) link
2. Tab till command bar present in top navigation section and press enter
3. Verify whether all options in command bar are visible or not

### Actual Result:
All options in command bar are not clearly visible at 400% zoom

### Expected Result:
All options in command bar should be clearly visible at 400% zoom. Entire content in the page should be clearly visible. There should be no loss of content

### User Impact:
Users with low vision who zoom the page for better visibility will find difficulty in accessing all the controls in command bar

### Attachment:
[A11y_Visual Studio Code_HomePage_Reflow.zip](https://github.com/microsoft/vscode-docs/files/6420888/A11y_Visual.Studio.Code_HomePage_Reflow.zip)

**Comment by kalpanah1** [comment]
#Closed;
Verified the issue in the below environment. The issue is fixed please find the attachment for reference.
URL: https://code.visualstudio.com.

**Comment by 12-shweta** [comment]
#A11yMAS;
